tests/migration_test.go

```diff
@@ -3537,6 +3539,75 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 			Expect(err).ToNot(HaveOccurred())
 		})
 	})
+
+	Context("On OpenShift", func() {
+		var vmi *v1.VirtualMachineInstance
+
+		BeforeEach(func() {
+			if !tests.IsOpenShift() {
+				Skip("Skip if not running on openshift")
+			}
+
+			By("creating the VMI")
+			vmi = cirrosVMIWithEvictionStrategy()
+			By("Starting the VirtualMachineInstance")
+			vmi = runVMIAndExpectLaunch(vmi, 240)
+		})
+
+		AfterEach(func() {
+			By("Deleting the VMI")
+			Expect(virtClient.VirtualMachineInstance(vmi.Namespace).Delete(vmi.Name, &metav1.DeleteOptions{})).To(Succeed())
+			By("Waiting for VMI to disappear")
+			tests.WaitForVirtualMachineToDisappearWithTimeout(vmi, 240)
+		})
+
+		getPDBForVMI := func(vmi *v1.VirtualMachineInstance) *v1beta1.PodDisruptionBudget {
+			pdbList, err := virtClient.PolicyV1beta1().PodDisruptionBudgets(util.NamespaceTestDefault).List(context.Background(), metav1.ListOptions{})
+			Expect(err).ToNot(HaveOccurred())
+
+			if len(pdbList.Items) < 1 {
+				return nil
+			}
+
+			for _, pdb := range pdbList.Items {
+				for _, ownerRef := range pdb.OwnerReferences {
+					if ownerRef.Name == vmi.Name {
+						alert := pdb.Labels[pdbs.OpenShiftPDBAtLimitAlert]
+						if alert == "disabled" {
+							return &pdb
+						}
+					}
+				}
+			}
+			return nil
+		}
+
+		It("[sig-compute]Created PDBs should silence the PodDisruptionBudgetAtLimit alert", func() {
```

Comment:
I don't think you need the `sig-compute` label since its already present in the top level spec container `Describe("[rfe_id:393]...`